src/main/java/com/conveyal/datatools/manager/models/Deployment.java

```diff
@@ -156,17 +156,25 @@ public Project parentProject() {
     /** If true, OSM extract will be skipped entirely (extract will be fetched from neither extract server nor URL. */
     public boolean skipOsmExtract;
 
+    /**
+     * If true, the EC2 deployment with otp-runner will be prepared to run an OTP jar file with the proper commands.
+     * NOTE: the {@link Deployment#otpVersion} is assumed to be properly set to an OTP2 jar file.
+     */
+    public boolean otp2;
```

Comment:
I don't really except 3.x to happen anytime soon, but I think it might be more future proof to use an enum here. It doesn't need to happen here, but using an enum for `tripPlannerVersion` would also allow let us get rid of the r5 boolean as well (not that we really intend to support that software moving forward).

Proposal:
- remove otp2 boolean
- add tripPlannerVersion (there's probably a better name) enum with `OTP_1.x` and `OTP_2.x` as the current options.

Reply:
Changes in #364

Reply:
proposed changes accepted